KonyAccelerate/userwidgets/tiles.ratingTile/modules/require/ratingTileController.js

```diff
@@ -1,6 +1,10 @@
 define(function() {
 
 	return {
-
+      setSpeakerProfileInRating:function(speaker){
```

Comment:
* ```setSpeakerProfileInRating: function(speaker) {```
* function comment is missing
* format with proper indentation

Reply:
Added Functional comments